src/pmpo_c.cpp

```diff
@@ -8,20 +8,20 @@
 namespace{
   std::vector<MPMesh_ptr> p_mpmeshes;////store the p_mpmeshes that is legal
     
-  void checkMPMeshValid(MPMesh_ptr p_mpmesh){
+  void checkMPMeshValid_f(MPMesh_ptr p_mpmesh){
```

Comment:
This should remain as 'checkMPMeshValid' (only functions in pmpo_c.h, and have C binding in pmpo_fortran.f90, should be appended with '_f')

Reply:
Thanks, Onkar, good catch. I miss that one.
Sina, if you really recompile the repo. The compiler should have an error message about this.